packages/paste-theme-utils/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@twilio-paste/theme-utils",
+  "version": "0.0.0",
+  "description": "Paste theme utilities for checking color contrast of Paste themes.",
```

Comment:
A few broad questions:

1) Any reason why we didn't put this into either the `design-tokens` or `theme` package?
2) Any plans to add more functionality to this? Right now it's focused on colors as per the description, which makes me wonder if this should be named something like `color-library`
3) Lastly, should this be a "library" package?

Reply:
OK, so yeah, I've been really struggling to find a home for these. It all started off in Design Tokens. Then I wanted it to be sharable with theme designer, and therefore based on `theme`, so it moved to Theme and the hook got created.

That added 12kb to theme for everyone, for no good reason. Why should the theme provider package suddenly ad 12k to your app because we want to test some themes in another internal app.

So then I pulled out of theme into it's own package, and I still kinda don't like it.

The fact that the utils require `GenericTokenShape` and `AllGenericTokens` from theme is problematic, as theme is a dependent of design tokens, so they can't live in Design tokens right now due to the circular dep. You could argue that those two should actually live in Design tokens though. 

I'm leaning towards utils living in Design Tokens, and the hook living in Theme as it's a hook to rate the current theme from theme provider.

Reply:
> I'm leaning towards utils living in Design Tokens, and the hook living in Theme as it's a hook to rate the current theme from theme provider.

That feels like a good solution.